x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/StickyTransactionProperties.tsx

```diff
@@ -91,5 +95,36 @@ export function StickyTransactionProperties({
     }
   ];
 
+  if (errorCount !== undefined) {
+    stickyProperties.push({
+      label: i18n.translate('xpack.apm.transactionDetails.errorsLabel', {
+        defaultMessage: 'Errors'
+      }),
+      val:
+        errorCount === 0 ? (
```

Comment:
This is a little hard to read. Can you split it up like `val: errorCount === 0 ? noErrorsText : errorsLink,`?